src/utils.js

```diff
@@ -146,12 +146,11 @@ function composeEventHandlers(...fns) {
 }
 
 /**
- * This generates a unique ID for all autocomplete inputs
- * @param {String} prefix the prefix for the id
+ * This generates a unique ID for an instance of Downshift
  * @return {String} the unique ID
  */
-function generateId(prefix) {
-  return `${prefix}-${idCounter++}`
+function generateId() {
+  return `${idCounter++}`
```

Comment:
nitpick: Since you'll be making a slight change anyway, I'd prefer this to be: `return String(idCounter++)` Mostly just because I like the explicit casting.

Thanks!